testing/firefox/APKBUILD

```diff
@@ -96,11 +96,19 @@ build() {
 	# set rpath so linker finds the libs
 	export LDFLAGS="$LDFLAGS -Wl,-rpath,${_mozappdir}"
 
+	case "$CARCH" in
+		x86_64)
+			arch_config="--disable-elf-hack"
```

Comment:
Why do this?

Reply:
I don't know what that flag does but it doesn't exist for aarch64 (it was just on the configure args list before, when only x86_64 was enabled), do you see a better way to do this?

Reply:
No, not really, but it'd be worth adding a comment to explain this here then :)